### PR TITLE
fix: use correct log prefixes

### PR DESCRIPTION
**Pull request by andrii-zakurenyi**
Use correct log prefixes to better communicate with the runner machine.

### CODE REVIEW

**Comment by andrii-zakurenyi** [comment]
I've tested it in my own lab :scientist: with code like this:
```
          echo "::debug My Debug message 1"
          echo "::debug:: My Debug message 2"
          echo "::debug::My Debug message 3"
          echo "::warning My Warning message 1"
          echo "::warning:: My Warning message 2"
          echo "::warning::My Warning message 3"
          echo "::error My Error message 1"
          echo "::error:: My Error message 2"
          echo "::error::My Error message 3"
```

And that's the result:
image
image

**Comment by andrii-zakurenyi** [comment]
btw, some of pre-commit hooks forces to update file generated by "changie":
```
Format YAML files........................................................Failed
- hook id: yamlfmt
- files were modified by this hook

.changes/unreleased/🐛 Bug Fix-20221026-222227.yaml  Done
```
:(

File: .changes/unreleased/🐛 Bug Fix-20221026-222227.yaml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+---
+kind: 🐛 Bug Fix
+body: use correct log prefixes which improves integration with GitHub Actions platform
```
Comment (sheldonhull) [inline .changes/unreleased/🐛 Bug Fix-20221026-222227.yaml:3]: @andrii-zakurenyi cc @mariiatuzovska  the comments here are specifically meant to communicate to the public, not developers. So it's a small difference, but think of this as a full description you'd send in an email or release notification to the wider audience. Adding suggested tweak for example.

We'll batch those up into a changelog when we are ready to do another release.

```suggestion
body: Adjust log prefix used in the GitHub action to improve the integration with actions and error identification.
```

**Review by sheldonhull — CHANGES_REQUESTED** [review]
See the check for linting is throwing up an issue. To bypass this you'll want to add a comment above the struct since it's not adhering to the kebab case default.

Use the [nolint directive](https://golangci-lint.run/usage/false-positives/#nolint-directive) with the specific linter + the name and reason. The format is as follows:

```go
// SecretToRetrieve should be documented.
type SecretToRetrieve struct { //nolint:tagliatelle // This json structure uses camelCase instead
}
}
```

That should solve any issues in the future for exceptions. All exceptions require a reason to be noted. For multiple issues then just `//nolint:unparam,cyclops,etc // Reasons`. 

Hope that helps. That should cut down this issue across all the structure fields.

**Comment by sheldonhull** [comment]
> btw, some of pre-commit hooks forces to update file generated by "changie":
> 
> ```
> Format YAML files........................................................Failed
> - hook id: yamlfmt
> - files were modified by this hook
> 
> .changes/unreleased/🐛 Bug Fix-20221026-222227.yaml  Done
> ```
> 
> :(

Yamlfmt should be excluded on that directory, but I updated a lot of files.
So you know how to fix, grab the equivalent exception from here: [trunk exclusion](https://github.com/DelineaXPM/dsv-cli/blob/84ec38a5a2ab3ecf102cc5ad3ed8259c1bdf61ca/.trunk/trunk.yaml#L59) that should get you situated.

If I get a chance I'll sync that across all the repos.

**Comment by sheldonhull** [comment]
> I've tested it in my own lab 🧑‍🔬 with code like this:
> 
> ```
>           echo "::debug My Debug message 1"
>           echo "::debug:: My Debug message 2"
>           echo "::debug::My Debug message 3"
>           echo "::warning My Warning message 1"
>           echo "::warning:: My Warning message 2"
>           echo "::warning::My Warning message 3"
>           echo "::error My Error message 1"
>           echo "::error:: My Error message 2"
>           echo "::error::My Error message 3"
> ```
> 
> And that's the result: image image

That's a great find. I didn't notice this failing and had removed the `::` at the end after reading the docs here which show you don't need that. [docs output an error](https://docs.github.com/en/actions/using-workflows/workflow-commands-for-github-actions#example-setting-an-error-message). 

Not sure why there's a disconnect there. As soon as we get the docs updated and linter fix we can merge it for sure.

**Comment by andrii-zakurenyi** [comment]
> See the check for linting is throwing up an issue. To bypass this you'll want to add a comment above the struct since it's not adhering to the kebab case default.
> 
> Use the [nolint directive](https://golangci-lint.run/usage/false-positives/#nolint-directive) with the specific linter + the name and reason. The format is as follows:
> 
> ```go
> // SecretToRetrieve should be documented.
> type SecretToRetrieve struct { //nolint:tagliatelle // This json structure uses camelCase instead
> }
> }
> ```
> 
> That should solve any issues in the future for exceptions. All exceptions require a reason to be noted. For multiple issues then just `//nolint:unparam,cyclops,etc // Reasons`.
> 
> Hope that helps. That should cut down this issue across all the structure fields.

I do not see any issues :thinking:

**Review by sheldonhull — APPROVED** [review]

**Pull request merged**